Cypress.on('uncaught:exception', (err, runnable) => {
    // Lidar com a exceção global aqui
  });
  